src/PaymentGateways/Gateways/ServiceProvider.php

```diff
@@ -109,13 +109,6 @@ private function registerGateways()
          * This module will eventually live in give-recurring
          */
         if (defined('GIVE_RECURRING_VERSION') && GIVE_RECURRING_VERSION) {
-            add_filter(
-                sprintf("givewp_gateway_%s_subscription_module", StripePaymentElementGateway::id()),
-                static function () {
-                    return StripePaymentElementGatewaySubscriptionModule::class;
```

Comment:
As we don't need the `StripePaymentElementGatewaySubscriptionModule` class here, we also can remove it from the imports at the top of this file. I noticed that the `TestOffsiteGateway` is also present in the imports but isn't used in the file, so it also can be removed.

Reply:
Resolved 84e4f780